scripts/shapella-upgrade/deploy-non-aragon-contracts-no-proxy-binding.js

```diff
@@ -0,0 +1,281 @@
+const runOrWrapScript = require('../helpers/run-or-wrap-script')
+const { log, logSplitter, logWideSplitter, yl, gr } = require('../helpers/log')
+const { readNetworkState, assertRequiredNetworkState } = require('../helpers/persisted-network-state')
+const { deployWithoutProxy, deployBehindOssifiableProxy, updateProxyImplementation } = require('../helpers/deploy')
+const { ZERO_ADDRESS, bn } = require('@aragon/contract-helpers-test')
+
+const { APP_NAMES } = require('../constants')
+
+const DEPLOYER = process.env.DEPLOYER || ''
+const TEMPLATE_PROXY_ADMIN = process.env.TEMPLATE_PROXY_ADMIN || ''
+const REQUIRED_NET_STATE = [
+  `app:${APP_NAMES.LIDO}`,
+  `app:${APP_NAMES.ORACLE}`,
+  "app:aragon-agent",
+  "app:aragon-voting",
+  "daoInitialSettings",
+  "oracleReportSanityChecker",
+  "burner",
+  "hashConsensusForAccounting",
+  "hashConsensusForValidatorsExitBus",
+  "withdrawalQueueERC721",
+]
```

Comment:
oracle daemon config?

Reply:
withdrawal vault

Reply:
why if we deploy them below and they do not accept any params?